src/sqlfluff/cli/commands.py

```diff
@@ -274,8 +274,13 @@ def dialects(**kwargs):
         "found. This is potentially useful during rollout."
     ),
 )
+@click.option(
+    "--not-ignore-files",
+    is_flag=True,
+    help=("If set, files will not be ignored by configuration files"),
```

Comment:
I find this a little confusing. Does this seem any better to you?

```suggestion
    help=("Perform the operation regardless of .sqlfluffignore configurations"),
```